README.md

```diff
@@ -73,6 +73,24 @@ our [blog post](https://blog.thunderbird.net/2023/07/k-9-mail-collaborates-with-
 
 You can report a security vulnerability [through the respective issues form](https://github.com/thunderbird/thunderbird-android/security/advisories/new).
 
+### Verifying the Apk Signature
+
+Users can ensure that the downloaded apk from Github and F-Droid was properly signed with our signing key by verifying
```

Comment:
I think we should be accurate with naming and replace "singing key" with "signing certificate". Especially since the command line argument is called `-printcert` and users might not be able to make the connection.

Reply:
I agree that "signing certificate" is more accurate, especially in the context of using the `keytool` command with `-printcert`. However, since this is about verifying the APK's signature, switching to the suggested term might not directly address the primary goal. It would be helpful to define the target audience for this documentation to determine whether the concern about users not making the connection is valid. That way, we can decide if a clarification or adjustment is truly necessary.

Reply:
I removed usage of any term to keep it simple

Reply:
`keytool` and `apksigner` verify that the APK was signed using the signing certificate that is part of the APK. What the user needs to do is verify that the signing certificate in the APK is actually our signing certificate and not some random other one.

Reply:
I understand your point, but it would be more helpful if you could suggest a way to explain this more clearly for users. That way, we can make sure the information is both accurate and user-friendly.